### PR TITLE
Adding collector SSL only

### PR DESCRIPTION
**Pull request by cassiodias**
* Defining environment variable enforcing "SSL only for collectors".
    * added config attribute with default value;
    * changed swagger definition;
    * added test.

### CODE REVIEW

File: config/collectorConfig.js [hunk 1]
```diff
@@ -31,6 +31,8 @@ if (heartbeatLatencyToleranceMillis < FLOOR) {
   heartbeatLatencyToleranceMillis = 1000 * LATENCY_DEFAULT;
 }
 
+const sslOnly = +pe.COLLECTOR_SSL_ONLY || false;
```
Comment (iamigo) [inline config/collectorConfig.js]: We use the "+" symbol when we want to try to coerce the env var to numeric value.
In this case here, we would want to set sslOnly to true if the env var **string** was "true" (and probably also accept "True"|"TRUE", but not "False" or "FALSE" or "false" ...

So I would suggest something like this:

```
const requireSslToRemoteDataSource = false;
if (pe.COLLECTOR_REQUIRE_SSL_TO_REMOTE_DATA_SOURCE && pe.COLLECTOR_REQUIRE_SSL_TO_REMOTE_DATA_SOURCE.toLowerCase() === 'true') {
  requireSslToRemoteDataSource = true;
}
```

Comment (iamigo) [reply config/collectorConfig.js]: Oh, we should also write this up in the README section "Securing Refocus" so that a Refocus admin knows how to do this.

Comment (cassiodias) [reply config/collectorConfig.js]: good point, thx

Comment (cassiodias) [reply config/collectorConfig.js]: Done

File: README.md [hunk 2]
```diff
@@ -62,6 +62,7 @@ See the [Quickstart](https://salesforce.github.io/refocus/docs/01-quickstart.htm
         the threshold of what we should consider "unused". Set the value
         to a negative time offset like `-30d` if you want to consider a
         token unused if it has not been used in the last 30 days.
+1. If you want Refocus Collectors collecting only from sources using HTTPS, set environment variable `COLLECTOR_REQUIRE_SSL_TO_REMOTE_DATA_SOURCE` to `true`.
```
Comment (iamigo) [inline README.md]: If you want to require SSL for connections from Refocus Collectors to the remote data sources, set environment variable `COLLECTOR_REQUIRE_SSL_TO_REMOTE_DATA_SOURCE` to `true`.

**Pull request merged**